### PR TITLE
image and input html

### PR DESCRIPTION
**Pull request by b0llu**
Added Image and Input to Documentation site
image
image
image
image
image

### CODE REVIEW

File: Components/Input/input.html [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+        sandbox="allow-scripts allow-same-origin">
+      </iframe>
+      <div class="divider"></div>
+      <h1 class="content-title">Placeholder Input</h1>
+      <p class="sub-content">Placeholder Input has a text present inside of the input field. They can be customized to
+        show error.
+        <div class="content-container">
+          <div class="input-group">
+            <label>Email</label>
```
Comment (mikesch803) [inline Components/Input/input.html:118]: label and input should be form

File: Components/Input/input.html [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+        </a>
+        <a href="https://twitter.com/TheBestDhruv">
+          <i class="fab fa-twitter icon"></i>
+        </a>
+        <a>
+          <i id="toggle-theme" class="fas fa-moon icon"></i>
+        </a>
+      </div>
+    </nav>
+    <div id="sidebar-menu" class="side-bar position-hidden">
```
Comment (Developers-codz) [inline Components/Input/input.html]: You should use semantic Html rather than using div everywhere like for sidebar use aside and main for main container

**Review by Developers-codz — APPROVED** [review]
Looks good to merge !

File: Components/Input/input.html (hunk 1 above)
Comment (b0llu) [reply Components/Input/input.html:118]: thanks for pointing it out i will change it asap!

File: Components/Input/input.html (hunk 2 above)
Comment (b0llu) [reply Components/Input/input.html]: thanks for pointing it out changing it asap!

**Review by partha8 — APPROVED** [review]
Looks good, go ahead with the merge.

**Pull request merged**